sematic-ai/sematic
# Torch lightning + Resnet + Ray example

Add an example that shows how you can use TorchLightning along with Sematic and Ray for distributed training and eval. Also illustrates our pipeline testing capabilities.

Testing
-------
- See [here](https://dev1.dev-usw2-sematic0.sematic.cloud/runs/66e59f3b9c074863bc69c16b3c49afb6) for fairly full training on 80 epochs, achieving ~85% accuracy
- See [here](https://dev1.dev-usw2-sematic0.sematic.cloud/runs/055f49a31e7347ec8c9d09b8bda328e7) to check docstring rendering (the latter was produced after the former, so the docstrings on 80 epochs are not complete)
